src/openrct2/drawing/Drawing.String.h

```diff
@@ -12,6 +12,7 @@
 #include "../core/FlagHolder.hpp"
 #include "../interface/Colour.h"
 #include "../interface/ColourWithFlags.h"
+#include "Drawing.h"
```

Comment:
Can we avoid this by replacing this with `TextColour.h`? I assume this is for the use of `TextColours`. I'd really like to avoid including `Drawing.h` further; it's already the cause for much recompilation.